napari/_qt/menus/file_menu.py

```diff
@@ -224,14 +224,14 @@ def _rebuild_samples_menu(self):
                     )
                     action = QAction(full_name, parent=self)
 
-                def _add_sample(*args, plg=plugin_name, smp=samp_name):
+                def _add_sample(*_, plg=plugin_name, smp=samp_name):
```

Comment:
Yes, tut it could be just `_` 
The triggered pass `bool` status to event action. 

I'm not too fond of this implementation as I prefer to use method and `self.sender()`, but this is out of the scope of this PR.